spec/11-statements.md

```diff
@@ -831,6 +831,7 @@ continue-statement:
 
 breakout-level:
   integer-literal
+  ( integer-literal )
```

Comment:
The correct syntax here is `'(' integer-level ')'`. Note that all other changes are autogenerated by tools/grammar.php.

It would be more accurate to use `'(' breakout-level ')'` (as `continue(((2)));` is also valid), though the way it actually works is that `breakout-level` is just `expression`, with a semantic Constraint that a literal integer is required.